### ISSUE TITLE
./f setwarp and /f addrule only lets certain players do it, regardless of rank

### ISSUE THREAD

**Issue by Momshroom** [MarkehMe/FactionsPlus#203]
I'm not sure what I've configured wrong.  I'm using the newest factions 2.8.7beta and the most recent factionsplus beta on Spigot 1.9.   
Who can do /f setwarp and /f addrule seems completely random, and unrelated to either faction rank or groupmanager rank.  Two different officers in the same faction:  one can and one cannot.  Another player is leader of their faction and cannot.   The same effect is true with rules.
These perms are assigned to the lowest rank in Groupmanager:
    - factionsplus.rules
    - factionsplus.chests
    - factionsplus.managewarps
    - factionsplus.setwarp
    - factionsplus.warp
Any ideas?
Thanks, and sorry for being a noob at this.

**Comment by MarkehMe** [comment]
Are you using any other plugins like FactionsPerms to change permission groups? 

Can you confirm the error message, as it should say if it was permission or rank

**Comment by Momshroom** [comment]
It says rank.

**Comment by MarkehMe** [comment]
Can you also confirm that there are no error messages in the console?

**Comment by Momshroom** [comment]
There aren't.

**Comment by MarkehMe** [comment]
[FactionsPlus.jar.zip](https://github.com/MarkehMe/FactionsPlus/files/187310/FactionsPlus.jar.zip)

I have changed the way recently with how I check these things (I use my own internal object rather than Bukkit's Player object).

Is there anything specific about your server? e.g. is it an offline server?

**Comment by Momshroom** [comment]
Yes, we're a bungeecord network so the factions server is in offline mode.

**Comment by Momshroom** [comment]
Thank you.  When I drop that jar in, I get this error:
http://pastebin.com/zJbUWtcK

**Comment by MarkehMe** [comment]
This might be where issues are coming from, as we check against UUIDs.. I will need to create some sort of internal UUID system for offline servers. Give me some time for this.

**Comment by MarkehMe** [comment]
[FactionsPlus.jar.zip](https://github.com/MarkehMe/FactionsPlus/files/187441/FactionsPlus.jar.zip)

Can you try this? This will detect a null UUID, and attempt to do something with it

**Comment by Momshroom** [comment]
Thank you so much for all the help.  I put the last jar in.  I've tried both as officer and as faction leader.  I don't get an error message trying to set a warp, but the console spams errors that look like this:
http://pastebin.com/rAsfydKd

Here are the pastebins from the error logs: 
http://pastebin.com/LswZ1hqM
http://pastebin.com/M7J0jXRE
When I try /f warps, I get:
"You must be in a faction to run this command!"
I confirmed that I was in a faction, and factions new it in both /f show and /f claim one

**Comment by MarkehMe** [comment]
To be honest I'm just having too many issues with 1.8.7.Beta and Minecraft 1.9, so I will just wait until there is a more stable version of Factions available :disappointed:

**Comment by Momshroom** [comment]
Okay.  Thank you very much for trying.  I really appreciate it.

**Comment by MarkehMe** [comment]
See #210
